baseline/OSPS-QA.yaml

```diff
@@ -62,31 +62,21 @@ criteria:
     security_insights_value: # TODO
 
   - id: OSPS-QA-03
-    maturity_level: 2
+    maturity_level: 1
     criterion: |
-      All released software assets MUST be
-      delivered with a machine-readable list of
-      all direct and transitive internal software
-      dependencies with their associated version
-      identifiers.
+      The code repository MUST contain a dependency 
+      list that accounts for the direct language dependencies
+      when the package management system supports it.
     rationale: |
       Provide transparency and accountability for
-      the project's dependencies, enabling users
-      and contributors to understand the
-      software's dependencies and versions.
+      the project's dependencies by fixing the desired versions
+      while enabling users and contributors to understand the
+      software's direct dependencies.
     details: |
-      This may take the form of a software bill of
-      materials (SBOM) or a dependency file that
-      lists all direct and transitive dependencies
-      such as package.json, Gemfile.lock, or
-      go.sum.
-
-      It is recommended to use a CycloneDX or SPDX
-      file that is auto-generated at build time by
-      a tool that has been vetted for accuracy.
-      This enables users to ingest this data in a
-      standardized approach alongside other
-      projects in their environment.
+      This may take the form a package manager or
+      language dependency lock file that ennumerates all
+      direct and, ideally, transitive build dependencies such as
+      package.json, Gemfile.lock, or go.sum.
```

Comment:
Do we include `requirements.txt` here despite the fact that it is not a lockfile, or no?

I don't have a strong preference, except that I'm going to write a policy to check this.

Reply:
`requirements.txt` _could_ be used to satisfy it as written, but doesn't necessarily. I do think there's a mismatch between the criterion and the rationale/details that I will raise as a separate suggestion.